### DISCUSSION TITLE
Pronúncia Correta

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#302] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/pronuncia-correta/

**Comment by brunoaugusto** [comment]
![null](<https://i.postimg.cc/wj9C0Q00/captain-america-i-understood-that-reference.png>)

**Comment by one-eyed-willy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [sábado, 30 de abril de 2022](<https://www.vish.blog.br/imagens/tirinhas/pronuncia-correta/#comment_1114006325>):

> [![null](<https://camo.githubusercontent.com/135a4573e7ced9b50fda5a65fd4ada8c7b6c13f1248fd1366dbeddfeae0caba0/68747470733a2f2f692e706f7374696d672e63632f776a3943305130302f6361707461696e2d616d65726963612d692d756e64657273746f6f642d746861742d7265666572656e63652e706e67>)](<https://camo.githubusercontent.com/135a4573e7ced9b50fda5a65fd4ada8c7b6c13f1248fd1366dbeddfeae0caba0/68747470733a2f2f692e706f7374696d672e63632f776a3943305130302f6361707461696e2d616d65726963612d692d756e64657273746f6f642d746861742d7265666572656e63652e706e67>)

![null](<https://i.postimg.cc/0jRkQJxF/twaog-the-game-ser-que-entendeu.png>)

**Comment by brunoaugusto** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [sábado, 30 de abril de 2022](<https://www.vish.blog.br/imagens/tirinhas/pronuncia-correta/#comment_1114007719>):

> [![null](<https://camo.githubusercontent.com/d1e03b2b7b52802fcd302ca42a7a80670788eddc6802985b760a96c5af77e7b8/68747470733a2f2f692e706f7374696d672e63632f306a526b514a78462f7477616f672d7468652d67616d652d7365722d7175652d656e74656e6465752e706e67>)](<https://camo.githubusercontent.com/d1e03b2b7b52802fcd302ca42a7a80670788eddc6802985b760a96c5af77e7b8/68747470733a2f2f692e706f7374696d672e63632f306a526b514a78462f7477616f672d7468652d67616d652d7365722d7175652d656e74656e6465752e706e67>)

Mas claro que entendi! É por causa mudança que o dublador Guilherme Briggs fez pra cena do episódio "A Trama da Glândula". Não sei se quem traduziu já tinha pensado na piada ou se foi improvisação, mas ficou sensacional



![null](<https://i.postimg.cc/6Q0ww5pL/Untitled-1.png>)

**Comment by angellisy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [sábado, 30 de abril de 2022](<https://www.vish.blog.br/imagens/tirinhas/pronuncia-correta/#comment_1114016126>):

> Mas claro que entendi! É por causa mudança que o dublador Guilherme Briggs fez pra cena do episódio "A Trama da Glândula". Não sei se quem traduziu já tinha pensado na piada ou se foi improvisação, mas ficou sensacional[![null](<https://camo.githubusercontent.com/8d1a5280b02ea3ad885e1f62bb987945b4b4a4b75dfc7bf6094d929aabf4c759/68747470733a2f2f692e706f7374696d672e63632f365130777735704c2f556e7469746c65642d312e706e67>)](<https://camo.githubusercontent.com/8d1a5280b02ea3ad885e1f62bb987945b4b4a4b75dfc7bf6094d929aabf4c759/68747470733a2f2f692e706f7374696d672e63632f365130777735704c2f556e7469746c65642d312e706e67>)

Guilherme Briggs é um dublador incrível

**Comment by tali-zorah** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [sábado, 30 de abril de 2022](<https://www.vish.blog.br/imagens/tirinhas/pronuncia-correta/#comment_1114016126>):

> Mas claro que entendi! É por causa mudança que o dublador Guilherme Briggs fez pra cena do episódio "A Trama da Glândula". Não sei se quem traduziu já tinha pensado na piada ou se foi improvisação, mas ficou sensacional[![null](<https://camo.githubusercontent.com/8d1a5280b02ea3ad885e1f62bb987945b4b4a4b75dfc7bf6094d929aabf4c759/68747470733a2f2f692e706f7374696d672e63632f365130777735704c2f556e7469746c65642d312e706e67>)](<https://camo.githubusercontent.com/8d1a5280b02ea3ad885e1f62bb987945b4b4a4b75dfc7bf6094d929aabf4c759/68747470733a2f2f692e706f7374696d672e63632f365130777735704c2f556e7469746c65642d312e706e67>)

Keelah, vão pra um quarto vocês dois
